package.json

```diff
@@ -44,7 +44,6 @@
     "mocha": "6.2.0",
     "mocha-junit-reporter": "^1.9.1",
     "nyc": "^14.1.1",
-    "source-map-support": "0.5.12",
```

Comment:
confirmed that this is not needed / is already connected automatically by ts-node